logging/google/cloud/logging/_helpers.py

```diff
@@ -46,3 +52,30 @@ def entry_from_resource(resource, client, loggers):
         return ProtobufEntry.from_api_repr(resource, client, loggers)
 
     raise ValueError('Cannot parse log entry resource.')
+
+
+def retrieve_metadata_server(metadata_key):
+    """Retrieve the metadata key in the metadata server.
+
+    See: https://cloud.google.com/compute/docs/storing-retrieving-metadata
+
+    :type metadata_key: str
+    :param metadata_key: Key of the metadata which will form the url.
```

Comment:
Documentation note: This is actually more than the key, at least as implemented, since you can also supply query parameters (E.g. "tags?alt=json" to change the output format).

This seems reasonable, I can't think of any security implications, so we should documented it.